Jenkinsfile

```diff
@@ -0,0 +1,77 @@
+node('linux && docker') {
+  checkout scm
+
+  withEnv([
+    'npm_config_cache=npm-cache'
+  ]){
+    withDockerContainer(image: 'node:14', args: '-u=root') {
+      stage('Install') {
+        sh 'yarn install'
+      }
+
+      stage('Build') {
+        sh '. ./read.version.sh'
+        sh 'echo $PACKAGE_JSON_VERSION'
+        sh 'yarn run injectTag'
+        sh 'yarn run build'
+
+        if (env.GIT_TAG_NAME) {
+          sh 'yarn run minimize'
+        }
+      }
+
+      stage('Install Chrome') {
+        sh "wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -"
+        sh "echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main' | tee /etc/apt/sources.list.d/google-chrome.list"
+        sh "apt-get update"
+        sh "apt-get install -y google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst fonts-freefont-ttf libxss1 libxtst6 --no-install-recommends"
+        sh "google-chrome --version"
+      }
+
+      stage('Test') {
+        sh 'yarn run unitTests'
+        
+        if (env.GIT_TAG_NAME) {
+          sh 'yarn run accessibilityTests'
+        }
+
+        sh 'set +e'
+        // sh 'yarn run uploadCoverage'
```

Comment:
I think the issue is the same as the one here: https://stackoverflow.com/questions/41978173/grunt-karma-coverage-fails-on-jenkins-but-works-on-windows-unix

I did not see a way to configure the option in the solution. I will revisit this in the future,